### DISCUSSION TITLE
Trying to install GHOST one click apps; getting 502

### DISCUSSION THREAD

**Discussion by gmcamposano** [caprover/caprover#1755] [Q&A]
So I have been looking for ways to get ghost running but seem to be getting:
NGINX 502 Error :/
If you are the developer, check your application's logs. See the link below for details

Logs show: 
2023-05-23T14:03:20.400764117Z exec /opt/bitnami/scripts/ghost/entrypoint.sh: exec format error
2023-05-23T14:03:26.117380487Z exec /opt/bitnami/scripts/ghost/entrypoint.sh: exec format error
2023-05-23T14:03:31.822095477Z exec /opt/bitnami/scripts/ghost/entrypoint.sh: exec format error
2023-05-23T14:03:37.474589845Z exec /opt/bitnami/scripts/ghost/entrypoint.sh: exec format error

Is there a way to install it?

**Comment by githubsaturn** [comment]
probably you're running on an arm server but  the ghost image is only built for x86

**Reply by gmcamposano** [reply]
Yes I am using an arm server. You are correct

**Reply by gmcamposano** [reply]
> probably you're running on an arm server but the ghost image is only built for x86

I tried to install it in a x86 droplet and its showing the same error.

**Reply by githubsaturn** [reply]
When you say "same error" - are you saying it's showing "exec format error" ?

**Reply by gmcamposano** [reply]
> When you say "same error" - are you saying it's showing "exec format error" ?

Sorry about not clarifying... 
NGINX 502 Error :/
If you are the developer, check your application's logs. See the link below for details

**Reply by githubsaturn** [reply]
Now check the app logs (where you used to see `exec format error`)  and see what's causing the issue. 

I just tried spinning up a ghost instance and it seems to be working just fine: 
http://test-ghost.server.demo.caprover.com/

You can login to the [demo](https://captain.server.demo.caprover.com/#/apps/details/test-ghost) instance using `captain42` password and see it for yourself.

![image](https://github.com/caprover/caprover/assets/10582082/058e0a7d-cf11-4897-94a6-f24a045b5cba)
